index.html

```diff
@@ -620,16 +620,20 @@ <h3>
           <dfn>Auto-releasing wake locks</dfn>
         </h3>
         <p>
-          A user agent may <a>release a wake lock</a> at any time it:
+          A user agent may <a>release a wake lock</a> at any time, for example,
+          when:
         </p>
         <ul>
-          <li>Detects abnormal operation: such as infinite loops and tasks
+          <li>It detects abnormal operation: such as infinite loops and tasks
           exceeding imposed time limits (if any).
           </li>
           <li>Battery is considered low and discharging.
           </li>
           <li>The user turns on some kind of device power conservation mode.
           </li>
+          <li>The operating system is configured to dim or turn off the screen
+          based on the user presence state.
```

Comment:
"user presence state" sounds very jargony... maybe something like?

```suggestion
          if the user is no longer physically present.
```

Or maybe "no longer within range of the device." ?

Reply:
This term is derived from https://en.wikipedia.org/wiki/Presence_information#Presence_state so I believe it should be fine for a web spec. Similar terminology is also used in e.g. XMPP presence protocol and some OS APIs and libraries.

There can be multiple contributing signals that make up this state. For example, a user might be sleeping in front of the screen and still be physically present and within the range, and so forth :-)

Reply:
Got it. Thanks for clarifying!